column_transforms/datediff/datediff.yaml

```diff
@@ -2,14 +2,14 @@ name: datediff
 description: Calculates the difference between two date, time, or timestamp expressions based on the date or time part requested. The function returns the result of subtracting the second argument from the third argument.
 arguments:
   date_part:
-    type: string
+    type: date_part
     description: |
        Must be one of the values listed in [Supported Date and Time Parts](https://docs.snowflake.com/en/sql-reference/functions-date-time.html#label-supported-date-time-parts)
   date_val_1:
-    type: string
+    type: mixed_value
```

Comment:
Are we OK with this just being a text field for the non-column input?

Reply:
for now, i think yeah, we can just specify in docs the format